frontend/src/container/MetricsApplication/MetricsPageQueries/OverviewQueries.ts

```diff
@@ -1,55 +1,150 @@
+import { OPERATORS } from 'constants/queryBuilder';
 import { BaseAutocompleteData } from 'types/api/queryBuilder/queryAutocompleteResponse';
 import { TagFilterItem } from 'types/api/queryBuilder/queryBuilderData';
-import { QueryBuilderData } from 'types/common/queryBuilder';
+import { DataSource, QueryBuilderData } from 'types/common/queryBuilder';
 
+import {
+	DataType,
+	FORMULA,
+	GraphTitle,
+	LETENCY_LEGENDS_AGGREGATEOPERATOR,
+	MetricsType,
+	OPERATION_LEGENDS,
+	QUERYNAME_AND_EXPRESSION,
+	WidgetKeys,
+} from '../constant';
 import {
 	getQueryBuilderQueries,
 	getQueryBuilderQuerieswithFormula,
 } from './MetricsPageQueriesFactory';
 
+export const letency = ({
+	servicename,
```

Comment:
serviceName should be a string here as backend would throw error when service name is undefined

Reply:
`serviceName` is coming from Params and can be undefined. So I think there should be a check of it.